blosc/Cargo.toml

```diff
@@ -13,13 +13,9 @@ categories = ["api-bindings"]
 keywords = ["compression"]
 
 [dependencies]
-blosc-sys = "1.14.4"
+blosc-sys = { version = "1.14.5-pre", path = "../blosc-sys" }
```

Comment:
The version number is based on whichever c-blosc library blosc-sys was built for.  If you dynamically generate bindings, then it's not possible to use this versioning scheme.

Reply:
So `1.14 ` corresponds to `blosc` 1.14? How do you deal with breaking changes in the `rust` bindings but not `blosc`?